include/mujincontrollerclient/mujincontrollerclient.h

```diff
@@ -79,6 +79,14 @@ class PlanningResultResource;
 class BinPickingResultResource;
 class DebugResource;
 
+/// \brief (scene) file entry in mujin controller
+struct FileEntry
+{
+    std::string filename;
+    double modified; // in epoch seconds
+    size_t size;
```

Comment:
@cielavenir is this the filesize? if so, can you add some comments like "file size in bytes".
perhaps you can also change the name to something more clear. size is very ambiguous.
thanks

Reply:
Oops, i see that you were copying webstack convention ;0/

in that case, just comment is fine.

Reply:
@rdiankov added comment